test/y2storage/pervasive_demo_test.rb

```diff
@@ -104,4 +112,35 @@ def zkey_output(file)
 
     include_examples "zkey cryptsetup actions"
   end
+
+  describe "#available?" do
+    let(:zkey_list_volume) { "" }
+
+    context "if secure key is available" do
+      let(:lszcrypt) { lszcrypt_output("ok") }
+
+      it "returns true" do
+        enc = blk_device.encrypt(method: pervasive)
+        expect(enc.method.available?).to be true
+      end
+    end
+
+    context "if no secure key devices are available" do
+      let(:lszcrypt) { lszcrypt_output("no_devs") }
+
+      it "returns false" do
+        enc = blk_device.encrypt(method: pervasive)
+        expect(enc.method.available?).to be false
+      end
+    end
+
+    context "if secure key is not supported" do
```

Comment:
I guess this part of the test is still not adapted to the usage of `locally!` (i.e. exception raised instead of `nil`).

Reply:
It does not mean the tool is not there. But the zcrypt module is not loaded (or so it said).

Reply:
`lszcrypt: error - cryptographic device driver zcrypt is not loaded!`